ts/features/walletV3/payment/store/actions/orchestration.ts

```diff
@@ -18,8 +23,13 @@ export const walletPaymentResetPickedPsp = createStandardAction(
   "WALLET_PAYMENT_RESET_PICKED_PSP"
 )();
 
+export const walletPaymentBackState = createStandardAction(
+  "WALLET_PAYMENT_BACK_STATE"
+)();
+
```

Comment:
What is the purpose of this action?

Reply:
Ops, I forgot to delete it. Addressed into 2102d662a244ebe8994df1c592a520945f0cd2fd